ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -114,7 +114,61 @@ import RevenueCat
         // See https://sdk.revenuecat.com/android/5.1.1/purchases/com.revenuecat.purchases/-purchases/-companion/can-make-payments.html
         return Purchases.canMakePayments()
     }
+}
+
+// MARK: Refund request
+@objc public extension CommonFunctionality {
+#if os(iOS)
+    @available(iOS 15.0, *)
+    @available(tvOS, unavailable)
+    @available(macOS, unavailable)
+    @available(watchOS, unavailable)
+    @available(macCatalyst, unavailable)
+    @objc(beginRefundRequestProductId:completionBlock:)
+    static func beginRefundRequest(productId: String,
+                                   completion: @escaping (ErrorContainer?) -> Void) {
+        guard let purchases = Self.sharedInstance as? PurchasesSwiftType else {
```

Comment:
`PurchasesSwiftType` already had completion block methods for these in `purchases-ios` but we are using the `PurchasesType` protocol in PHC. I'm guessing the separation between `PurchasesType` and `PurchasesSwiftType` was made so people could use purchases from ObjC without any issues. I'm not sure if there is a better way to access those methods though, and I thought we should use them rather than write another implementation in PHC. Lmk if you have other ideas though!